frigate/app.py

```diff
@@ -734,12 +655,6 @@ def start(self) -> None:
         self.start_watchdog()
         self.init_auth()
 
-        # Flask only listens for SIGINT, so we need to catch SIGTERM and send SIGINT
```

Comment:
`SIGINT` works in the devcontainer (ctrl-c), but Docker sends `SIGTERM`. Is this no longer necessary?

Reply:
I'm not sure just yet to be honest. I'd like to unravel the multiprocessing mess first.

Currently frigate never really shuts down cleanly (it always hits the 10 second timeout in docker). So until I can get the devcontainer to shut down in under a second, it's hard to test. I _think_ it's unnecessary, but I'd rather come back to it later.

PS: What I mean by "mess" is to try running with `mp.set_start_method("spawn", force=True)`. (Here's the reason [why](https://github.com/python/cpython/issues/84559)). We're hitting all kinds of undefined behaviors right now.

Reply:
with a docker timeout of 30 seconds Frigate shuts down cleanly before the timeout for me so I think it would be good to ensure that the behavior stays the same after this PR is merged. There is another user contributing a separate change that will likely remove it anyway

Reply:
Huh. It does stop, but it takes ~120 seconds for me (vs ~30 in 0.14). This needs some investigation then.

Reply:
Okay. As far as the signal handler goes, I say we leave it here. Since there's another change on the way that might remove it anyway, might as well leave it for then.

But I'll leave the final call to you.

Reply:
agreed, leave it in for now

Reply:
It's in __main__.py

Reply:
Actually we probably need to habdle SystemExit in addition to keyboardinterrupt now. Give me a bit to fix that.

Reply:
Done.